src/domUpdates.js

```diff
@@ -71,6 +72,21 @@ let domUpdates = {
     }
   },
 
+  displayCurrentTrip: () => {
+    const currentTrip = document.querySelector('.current-trip');
+    const welcome = document.querySelector('.welcome');
+    domUpdates.traveler.findCurrentTrip();
+    if (domUpdates.traveler.currentTrip === undefined) {
+      welcome.innerText = `Welcome Home! Ready to Wander?`;
+    } else {
+      let trip = new Trip(domUpdates.traveler.currentTrip)
+      let name = domUpdates.destinations[trip.destinationID - 1].destination;
+      let image = domUpdates.destinations[trip.destinationID - 1].image;
+      welcome.innerText = `Welcome to ${name}`;
+      currentTrip.insertAdjacentHTML('beforeend', `<img class='trip-picture' src='${image}' alt='Photo here'><br><h4 class="current-display">Date: ${trip.date}<br>Duration: ${trip.duration} days<br>Traveling Party: ${trip.travelers} people</h4>`)
```

Comment:
I think to get full points for accessibility we will need to interpolate the alt key from the destination into this line's alt tag. But I think that should be a quick fix, it seems like you already have access to that data. Pretty much the same thing you're doing on 84 but with the alt key. Also, I hate how `beforeend` is not camelCase. Boo to the JS language dev who decided that was okay.

Reply:
Thanks for this. It was late and I was trying this to display the image and see if my logic was sound with the intention to come back and fix the alt tag if it did. I forgot to do that before the commit/pr. Added to my refactor list.

Reply:
Done!